example/tailor.js

```diff
@@ -5,8 +5,9 @@ const Tailor = require('../index');
 const fetchTemplateFs = require('../lib/fetch-template');
 const serveFragment = require('./fragment');
 const baseTemplateFn = () => 'base-template';
-
+const AMD_LOADER = 'file://' + require.resolve('iamdee');
```

Comment:
If someone was relying on a full featured require.js this is a breaking change, I would keep it as a default for now.